### PR TITLE
fix(gui): contain Integrity raw diagnostic detail overflow

### PR DESCRIPTION
**Pull request by harishkamathuk**
## Summary
- finish the remaining #136 Integrity overflow fix by containing the raw diagnostic/error detail surface
- preserve full diagnostic text while preventing horizontal bleed
- keep the change local to the remaining problematic field

## What changed
- updated the `signal.details` surface in `IntegrityPage` to render inside a bounded raw-text block
- applied:
  - `pre`
  - `overflow-x-auto`
  - `whitespace-pre-wrap`
  - `[overflow-wrap:anywhere]`
  - `break-words`
  - monospace + muted/bordered styling
- left unrelated Integrity fields unchanged

## Scope protection
Did not change:
- backend/API behavior
- wider Integrity workflow
- unrelated fields or page redesign

## Validation
`cd operator_console/gui_app && npm run test -- src/test/integrity-page.test.tsx`

## Related
#136

---

<a href="https://app.devin.ai/review/harishkamathuk/media-manager/pull/149" target="_blank">
  <picture>
    <source media="(prefers-color-scheme: dark)" srcset="https://static.devin.ai/assets/gh-open-in-devin-review-dark.svg?v=1">
    Open in Devin Review
  </picture>
</a>

### CODE REVIEW

File: operator_console/gui_app/src/pages/IntegrityPage.tsx [hunk 1]
```diff
@@ -661,7 +661,7 @@ export default function IntegrityPage() {
                         <p className="font-medium">{signal.signal_type}</p>
                       </div>
                       <p className="mt-1 text-xs uppercase tracking-wide text-muted-foreground">{signal.severity}</p>
-                      <pre className="mt-2 overflow-x-auto rounded-lg bg-muted p-3 text-xs">
+                      <pre className="mt-2 overflow-x-auto whitespace-pre-wrap break-words [overflow-wrap:anywhere] rounded-lg border border-border/60 bg-muted/40 p-3 font-mono text-xs">
```
[the remark this answers is not in the document]
Comment (harishkamathuk) [reply operator_console/gui_app/src/pages/IntegrityPage.tsx:664]: Confirmed. Wrapping is now the primary containment strategy for this raw diagnostic surface, so `overflow-x-auto` is mostly a defensive fallback rather than the main behavior. I’m comfortable leaving it in place here as a harmless safety net for edge-case raw co

**Comment by harishkamathuk** [comment]
Verified against the current test. This is a fair maintainability nit, but I’m leaving the assertions as-is in this PR because the exact containment classes are the mechanism of the fix and the test is intentionally checking that chosen pattern. If we want to relax these to more behavioral assertions later, that would be a small follow-up rather than part of this scoped overflow fix.
